api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -275,4 +280,19 @@ public void removeUserFromGroup(String email, String groupName) {
       return null;
     });
   }
+
+  @Override
+  public boolean isUserMemberOfGroup(String groupName) {
+    return retryHandler.run((context) -> {
+      // There is no endpoint in FireCloud for checking whether a user is a member of a particular
+      // group; so instead, fetch all the group memberships. (There won't be that many for our
+      // users anyway.)
+      for (ManagedGroupAccessResponse group : endUserGroupsApiProvider.get().getGroups()) {
+        if (group.getGroupName().equals(groupName) && group.getRole().equals("member")) {
```

Comment:
I'd probably reverse the `.equals()` ordering here to reduce risk of NPEs (obvious for the constant, but I'd also say the API response is less trusted than the function parameter `groupName`).

Reply:
Done.